ci/scripts/build-project.sh

```diff
@@ -5,5 +5,14 @@ source $(dirname $0)/common.sh
 repository=$(pwd)/distribution-repository
 
 pushd git-repo > /dev/null
-./gradlew -Dorg.gradle.internal.launcher.welcomeMessageEnabled=false --no-daemon --max-workers=4 -PdeploymentRepository=${repository} build publishAllPublicationsToDeploymentRepository
+if [[ -d /opt/openjdk-secondary ]]; then
```

Comment:
I'm not super happy with this block here, but as said: this can be cleaned up when there is a Gradle version that works with JDK 14